tools/issue-labeler/src/IssueLabelerService/AzureSdkIssueLabelerService.cs

```diff
@@ -14,62 +13,242 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
+using AzureRAGService;
+using Azure.Identity;
+using System.Linq;
 
 namespace IssueLabelerService
 {
     public class AzureSdkIssueLabelerService
     {
-        private static readonly ActionResult EmptyResult = new JsonResult(new PredictionResponse(Array.Empty<string>()));
+        private static readonly ActionResult EmptyResult = new JsonResult(new IssueOutput { Labels = [], Answer = null, AnswerType = null });
```

Comment:
What we'll want to do in here is refactor such that the Function holds only the following responsibilities:
    - Understand config
    - Reach out to some label predictor that lives elsewhere (old or AI, based on config)
    - Evaluate label results
    - Reach out (or don't) to some answer predictor that lives elsewhere.

The label predictor and answer predictors should be concrete classes that we create based on config and which live in some static member or get injected via DI.

The Function is just the request/response shell that knows how to talk to everyone and what questions to ask.  It has no understanding of how labels or answers are predicted - just the contract.